lib/prepare_signing_flow.py

```diff
@@ -110,7 +112,22 @@ def prepare_signing_flow(sh_session: SigningHubSession, sign_flows: List[Dict]):
             piece_uri = sign_flow["piece"]
 
             # Document
-            signinghub_document_uri, _, _ = upload_piece_to_sh(piece_uri, package_id)
+            signinghub_document_uri, _, signinghub_document_id = upload_piece_to_sh(piece_uri, package_id)
+
+            if sign_flow["decision_report"]:
```

Comment:
This isn't going to work. In the endpoint route we set this field to `None` for beslissingsfiches: https://github.com/kanselarij-vlaanderen/digital-signing-service/blob/master/web.py#L63

When the piece that's going to be signed has the same URI as its agendaitem's BF, we nullify the field so that we don't add the BF as a voorblad. I think you've branched off of old code and that's why you don't see that, make sure to merge master into your branch.

Also just checking for that field won't necessarily work we can sign pieces for agendaitems with no BF, we also don't want to place signature fields on those. You really need to check the type here.

Reply:
Ah, I do have an older version, it was working on my end. Will fix.